docs/DeveloperGuide.adoc

```diff
@@ -782,13 +782,15 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
+* <<member>> of a <<club>>
+* has a need to manage a significant number of members
 * prefer desktop apps over other types
 * can type fast
 * prefers typing over mouse input
 * is reasonably comfortable using CLI apps
+* has a need to communicate or notify members
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*: efficiently manage members faster than a typical mouse/GUI driven app
```

Comment:
Wouldn't it be better if our value proposition focused on 'efficiently managing clubs' as a whole? We have thought of features related to tasks and polls which are directly related to club activities - so we're not just managing club members.

Reply:
Yeah, sure. Fell free to update it as you feel necessary. I was just trying to get us started.